### ISSUE TITLE
Won't update from 1.3.5

### ISSUE THREAD

**Issue by sampSCS** [bruderstein/nppPluginManager#22]
Plugin Manager says that version 1.4.3 is available.  

When I try to update to this version from 1.3.5, everything seems to install correctly, and then I get a dialog that says a restart is required to complete the installation, but after the restart, Plugin Manager still shows the installed version as 1.3.5.

I have tried running Notepad++ as administrator, but it didn't change anything.  My domain login is a member of the local administrators group.

There are 3 other plugins that seem to have the same problem, Customize Toolbar, Python Script, and XML Tools.  They all require restarts to complete, but after Notepad++ restarts, they also show the same installed version as prior to the update.

What can I do to resolve this problem?

**Comment by vlakoff** [comment]
Did you accept the UAC prompts? I have just updated, and got 2 of them.

**Comment by Przbl** [comment]
Same here with the dialog.
I copied the plugin manager files manually: the dialog remains, but when I click 'cancel', np++ opens and the plugin manager presents itself updated. Nevertheless do I get the same nasty dialog on each start. When I click 'update' something is downloaded and then np++ hangs, reporting the gpup.exe process. I have to kill the task to go on working, so I ended up deleting the PluginManager.dll and the gpup.exe. Now the errors are gone but there's no plugin manager anymore.

**Comment by sampSCS** [comment]
@vlakoff : UAC has been turned off on this computer, there were no prompts.

@Przbl : If you are referring to the "There's new update(s) available" dialog, you can turn that off through the Plugin Manager Settings.

If you are referring to the "restart required to finish update" dialog that you get after performing the update through Plugin Manager itself, then I'm going to guess that it was your manually updating the files after Plugin Manager had already scheduled the "update" that caused your problem.  The "restart" dialog will reappear every time you start NPP until you click yes, and when you finally did, the reason it hung might have something to do with it checking for (and not finding) the old version of the files it was expecting to overwrite.... if that makes any sense?

I've thought about removing/re-installing Plugin Manager in an attempt to resolve this problem, but was hoping someone had a better idea.

**Comment by bruderstein** [comment]
I've tried to reproduce this, and sadly I can't. What I would suggest is to delete the file `PluginManagerGpup.xml` from `%APPDATA%\Notepad++\plugins\config` (assuming you've got a standard installation - if you're on a portable installation, it'll just be in your Notepad++ directory under `plugins\config`), and then restart Notepad++. 

It sounds like there's something got stuck in this file.  It would be really really helpful if you could post the contents of this file before deleting it (there can be absolute paths in there, so feel free to "redact" those, and just leave the filenames, or send to me privately).

**Comment by Przbl** [comment]
When I deleted the PluginManagerGpup.xml I could start np++ without problems.
After opening the plugin manager and trying to update it to the new version the same error occurred again.
There was a new PluginManagerGpup.xml created that only had this content:
```
<install>
    <delete file="D:\Notepad++\plugins\PluginManager.dll" />
    <copy from="D:\Notepad++\plugins\Config\plugin_install_temp\plugin1\plugins\PluginManager.dll" to="D:\Notepad++\plugins\" replace="true" />
</install>
```
So I killed all np++-tasks, copied the two new plugin manager files manually and deleted the PluginManagerGpup.xml. Now the problem is gone and I can start np++ normally and use the new plugin manager. Still no idea where the problems came from.
